### ISSUE TITLE
Feature request: automatically enable all repositories for an organization

### ISSUE THREAD

**Issue by wouterboomsma** [travis-ci/travis-ci#7828]
I'm trying to set up travis for teaching a programming course, similar to what is described here:
    https://github.com/blog/2324-real-time-feedback-for-students-using-continuous-integration-tools
The short story is that GitHub Classroom allows you to send out an invitation to students, which, when accepted, will automatically setup a new repository for that student for the current assignment. The student can push to this repository to hand in the assignment, and travis can then be set up to automatically run unit tests to check the correctness of the assignment.

One stumbling block in this context is that new repositories are not automatically enabled in Travis, so for each of the >100 students, I need to manually flick the switch. This makes it difficult to give them real-time feedback on their progress with the assignment. It would be absolutely fantastic if there was a way of setting the default activity state for repositories within an organization to "On", so that new assignment repositories would automatically be activated upon creation.

**Comment by timofonic** [comment]
@wouterboomsma Yes, I need that. I love to tinker with many Open Source projects and manually enabling them is a total PITA. Are there some greasymonkey script as a workaround at least?

Not only that, but it's even worse when you have to enable to a big number of organizations...

@BanzaiMan Any news?

**Comment by BanzaiMan** [comment]
@timofonic Sorry, no news to share at this time.
